src/sql/workbench/parts/notebook/cellViews/outputArea.css

```diff
@@ -14,4 +14,12 @@ output-area-component .notebook-output {
 
 .output-userselect.actionselect {
 	user-select: text;
+}
+
+.output-userselect pre{
+	white-space: pre-wrap;       /* Since CSS 2.1 */
```

Comment:
Do we need all of these? We only care about chromium, right?

Reply:
yep, you're right. Am so used to adding for web I copied all. :(